content/install_maintain/installation/installing-cluster.md

```diff
@@ -144,19 +147,26 @@ The PostgreSQL database high-availability cluster is comprised of 3 nodes (Cloud
  tcp/2380  | etcd port.
  tcp/5432  | PostgreSQL connection port.
  tcp/8008  | Patroni control port.
+ tcp/8009  | Monitoring service port.
```

Comment:
This should also be updated in https://docs.cloudify.co/latest/install_maintain/installation/prerequisites/
and in https://docs.cloudify.co/latest/install_maintain/installation/installing-cluster/